src/battle_message.c

```diff
@@ -231,6 +231,7 @@ static const u8 sText_PkmnBlewAwaySpikes[] = _("{B_ATK_NAME_WITH_PREFIX} blew aw
 static const u8 sText_PkmnFledFromBattle[] = _("{B_ATK_NAME_WITH_PREFIX} fled from\nbattle!");
 static const u8 sText_PkmnForesawAttack[] = _("{B_ATK_NAME_WITH_PREFIX} foresaw\nan attack!");
 static const u8 sText_PkmnTookAttack[] = _("{B_DEF_NAME_WITH_PREFIX} took the\n{B_BUFF1} attack!");
+static const u8 sText_FutureSightNoHit[] = _("{B_CURRENT_MOVE} did not hit because\nthe target is fainted.");
```

Comment:
Is this authentic game text? The tone just feels more Showdowny than most other strings to me, but that could just be me haha

Reply:
That is from showdown lol Idk the offical game freak string and it is is different.

Reply:
It's missing from the SV textdump, so we should really try to test this in-game before we add a non-vanilla string imo (unless others think it's non-blocking). This PR is good to go other than this though

Reply:
I removed the string since there wasn't this are any other comparable string in the SV data. I didn't replace it with anything else because the `But failed` string seemed weird in this situation and since the set up is non trivial I don't want to bother anyone. Though it might worth an issue ticket. Maybe someone will test it.

Reply:
fixed